server/index.js

```diff
@@ -25,6 +37,45 @@ app.get('/app.js', cors(), async (req, res) => {
   res.sendFile(path.join(__dirname, '../public/dist/bundle.js'))
 });
 
+//CRUD routes
+//create new user
+app.post('/api/users', (req, res) => {
+  let userName = req.body.name;
+  User.insertMany(fakeUser(userName))
+    .then(data => console.log(`Inserted ${data} into database`))
+    .then(res.sendStatus(200))
+    .catch(err => console.log(`ERROR createing user: ${err}`));
+})
```

Comment:
Clear, simple, and easy to follow.
Curious why insertMany over a single method like .save?
Recommend dropping the console log for success once you're confident this works. Could send a success message to client?
Also recommend changing the error entry to console.error
Nice use of backticks and string templates!